src/main/java/org/toradocu/translator/Matcher.java

```diff
@@ -349,4 +368,31 @@ private static String simpleMatch(String predicate) {
     }
     return predicateTranslation;
   }
+
+  private static String isNumberWord(String numberString) {
```

Comment:
I suggest a better naming for this function to make its purpose more clear.
Something like `numberWordToDigit` can work.